<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvPrintCustomLayout</class>
 <widget class="QDialog" name="CustomLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>221</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>221</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Custom layout</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="m_photoGridCheck">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>46</height>
          </size>
         </property>
         <property name="text">
          <string>Photo grid</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_fitAsManyCheck">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>46</height>
          </size>
         </property>
         <property name="text">
          <string>Fit as many as possible</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_photosXPageCheck">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>46</height>
          </size>
         </property>
         <property name="text">
          <string>Photos per page</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>18</height>
              </size>
             </property>
             <property name="text">
              <string>Rows</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="m_gridRows">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>18</height>
              </size>
             </property>
             <property name="text">
              <string>Columns</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="m_gridColumns">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>Photo size</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="m_photoHeight">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="m_photoWidth">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_photoUnits">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>cm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>inches</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>18</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="m_photosXPage">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="specialValueText">
              <string extracomment="0 to disable"/>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="m_autorotate">
       <property name="text">
        <string>Auto rotate</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>258</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_doneButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_photosXPageCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_photosXPage</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fitAsManyCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_photoHeight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fitAsManyCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_photoWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fitAsManyCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_photoUnits</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_photoGridCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_gridRows</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_photoGridCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_gridColumns</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_photosXPageCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_autorotate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
